docs/v3-LIMITS.md

```diff
@@ -7,6 +7,8 @@ Known differences
 
 - On the upside, there are no forbidden headers.
 
+- Passing basic authentication credentials (`https://username:password@hostname`) in the URL is supported.
```

Comment:
For now lets just throw a TypeError if url includes credentials in the Request constructor.
We can decide later what to do with it. Want to find out what the real reason is first.